<launch>

  <node pkg="waypoint_trajectory_generator" type="trajectory_generator_node" name="trajectory_generator_node" output="screen" required = "true">

      <remap from="~waypoints" to="/waypoint_generator/waypoints"/>
      <remap from="~odom"      to="/vicon_imu_ekf_odom"/>
      <remap from="~joy"      to="/nouse"/>
      
      <param name="planning/vel"       value="1.5"  />
      <param name="planning/acc"       value="5.0"  />
      <param name="planning/dev_order" value="4"    />
      <param name="planning/min_order" value="3"    />
      <param name="vis/vis_traj_width" value="0.06" />

  </node>

  <node pkg="waypoint_generator" name="waypoint_generator" type="waypoint_generator" output="screen">        
      <remap from="~goal" to="/goal"/>
      <param name="waypoint_type" value="manual"/>    
  </node>

  <node name="rvizvisualisation" pkg="rviz" type="rviz" output="log" args="-d $(find waypoint_trajectory_generator)/launch/rviz_config/test_traj.rviz" />

  <!--include file="$(find traj_server)/launch/traj_server.launch">
  </include-->

  <include file="$(find odom_visualization)/launch/odom_visualization.launch">
  </include>

</launch>
